qiskit/circuit/register.py

```diff
@@ -130,6 +130,17 @@ def size(self):
         """Get the register size."""
         return self._size
 
+    def index(self, element):
```

Comment:
Is this function used anywhere? If not we should remove it from this PR.

Reply:
I thought I would need it in this pr but thought it would be generally useful. I can remove it from this pr though.